cli/src/commands/extensions/api.rs

```diff
@@ -233,13 +235,14 @@ async fn get_package_details(
 /// Parse a lockfile and return the package descriptors contained therein.
 /// Equivalent to `phylum parse`.
 #[op]
-fn parse_lockfile(lockfile: &str, lockfile_type: &str) -> Result<Vec<PackageDescriptor>> {
+fn parse_lockfile(lockfile: String, lockfile_type: String) -> Result<Vec<PackageDescriptor>> {
     let parser = LOCKFILE_PARSERS
         .iter()
         .find_map(|(name, parser)| (*name == lockfile_type).then(|| *parser))
         .ok_or_else(|| anyhow!("Unrecognized lockfile type: `{lockfile_type}`"))?;
 
-    let lockfile_data = std::fs::read_to_string(Path::new(lockfile))?;
+    let lockfile_data = fs::read_to_string(&lockfile)
```

Comment:
Hm I suppose we could, but that's fundamentally unrelated to this PR.

Did you have any reason for initially designing it as synchronous? Was it an oversight or just because of the existing implementation in use by subcommands?

Reply:
Yeah, it is unrelated, just something I noticed while reviewing. I guess I just did not see an immediate need for it at the time and it did not occur to me to have this be `async`.